tests/settings.py

```diff
@@ -0,0 +1,145 @@
+"""
+Django settings for temp project.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/stable/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/stable/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(PROJECT_DIR, ...)
+PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
+BASE_DIR = os.path.dirname(PROJECT_DIR)
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/stable/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "c6u0-9c!7nilj_ysatsda0(f@e_2mws2f!6m0n^o*4#*q#kzp)"
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = ["localhost", "testserver"]
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    "wagtail_footnotes",
+    "tests",
+    "wagtail.contrib.search_promotions",
+    "wagtail.contrib.forms",
+    "wagtail.contrib.redirects",
+    "wagtail.embeds",
+    "wagtail.users",
+    "wagtail.snippets",
+    "wagtail.documents",
+    "wagtail.images",
+    "wagtail.search",
+    "wagtail.admin",
+    "wagtail.api.v2",
+    "wagtail.contrib.modeladmin",
+    "wagtail.contrib.routable_page",
+    "wagtail.contrib.styleguide",
```

Comment:
question: why include the api, routable_page, styleguide, redirects, search_promotions, embeds, Django sitemaps, forms? if the app only looks at a custom StreamField (with rich text)